@xen-orchestra/vmware-explorer/esxi.mjs

```diff
@@ -58,6 +58,9 @@ export default class Esxi extends EventEmitter {
     const res = await this.search('Datacenter', ['name', 'datastore'])
     await Promise.all(
       Object.values(res).map(async ({ datastore, name }) => {
+        if (datastore.ManagedObjectReference === undefined) {
+          return
+        }
```

Comment:
Don't you think a log might be useful?

Reply:
no, it can be completly acceptable ( in the ticket it is a datastore that is phased out, since the vm are migrated to xcp)
An error will be thrown if XO try to read from it